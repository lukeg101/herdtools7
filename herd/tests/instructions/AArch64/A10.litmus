AArch64 A10


 { int64_t x; int64_t 0:x0=1; 0:x2=x}

P0;
  STR X0, [X2];

exists (x=1)
