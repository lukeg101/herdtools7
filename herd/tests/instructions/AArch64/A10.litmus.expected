Test A10 Allowed
States 1
x=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=1)
Observation A10 Always 1 0
Hash=ea5191f717adc74412c7fe1dae4db891

